code/datums/uplink/medical.dm

```diff
@@ -6,36 +6,36 @@
 
 /datum/uplink_item/item/medical/sinpockets
 	name = "Box of Sin-Pockets"
-	desc = "A box of filled dough pockets. Great for a quick meal when you're hiding from Security. Instructions included on the box."
+	desc = "A box of filled dough pockets. Great for a quick meal. Instructions included on the box."
 	item_cost = 8
 	path = /obj/item/storage/box/sinpockets
 
 /datum/uplink_item/item/medical/stabilisation
 	name = "Stabilisation First Aid Kit"
 	desc = "Contains variety of emergency medical pouches."
-	item_cost = 16
+	item_cost = 4
 	path = /obj/item/storage/firstaid/stab
 
 /datum/uplink_item/item/medical/stasis
 	name = "Stasis Bag"
 	desc = "Reusable bag designed to slow down life functions of occupant, especially useful if short on time or in a hostile enviroment."
-	item_cost = 24
+	item_cost = 12
 	path = /obj/item/bodybag/cryobag
 
 /datum/uplink_item/item/medical/defib
 	name = "Combat Defibrillator"
 	desc = "A belt-equipped defibrillator that can be rapidly deployed. Does not have the restrictions or safeties of conventional defibrillators and can revive through space suits."
-	item_cost = 24
+	item_cost = 12
 	path = /obj/item/defibrillator/compact/combat/loaded
 
 /datum/uplink_item/item/medical/surgery
 	name = "Surgery Kit"
 	desc = "Contains all the tools needed for on the spot surgery, assuming you actually know what you're doing with them. Floor sterilization not included."
-	item_cost = 40
+	item_cost = 8
```

Comment:
that is stupid low

Reply:
40 out of 120 telecrystals is stupid high as well